### PR TITLE
Homework_6_Olha_Lesiuk

### PR DESCRIPTION
**Pull request by OlhaLesiuk**

### CODE REVIEW

File: Olha_Lesiuk/HW_6/task_1.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import os
+from random import randint
+a_left_operand = []
+b_right_operand = []
+c_operations = []
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_6/task_1.py]: you do not ned a,b,c in naming

File: Olha_Lesiuk/HW_6/task_2.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    if elements[i][2] == "1":
+        print(int(elements[i][0]) + int(elements[i][1]))
+    elif elements[2] == "2":
+        print(int(elements[i][0]) - int(elements[i][1]))
+    elif elements[2] == "3":
+        print(int(elements[i][0]) * int(elements[i][1]))
+    else:
+        print(int(elements[i][0]) / int(elements[i][1]))
+
+print(output_of_results)
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_6/task_2.py:18]: image
Please fix it

File: Olha_Lesiuk/HW_6/task_3.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+if __name__ == '__main__':
+    with open("text.txt", "r") as file:
+        frequency_text = file.read()
+
+        for letter in set(frequency_text.lower()):
+            if letter in "abcdefghijklmnopqrstuvwxyz":
```
Comment (bohdandovbsyh) [inline Olha_Lesiuk/HW_6/task_3.py:7]: you can use
import sting
alphabet = string.ascii_lowercase

Comment (OlhaLesiuk) [reply Olha_Lesiuk/HW_6/task_3.py:7]: should I edit it? or do you approve current solution?

File: Olha_Lesiuk/HW_6/task_2.py (hunk 2 above)
Comment (OlhaLesiuk) [reply Olha_Lesiuk/HW_6/task_2.py:18]: Fixed

File: Olha_Lesiuk/HW_6/task_3.py (hunk 3 above)
Comment (bohdandovbsyh) [reply Olha_Lesiuk/HW_6/task_3.py:7]: Just for future

**Review by bohdandovbsyh — APPROVED** [review]

**Pull request merged**